index.bs

```diff
@@ -1275,20 +1422,26 @@ To <dfn>trigger event-level attribution</dfn> given an [=attribution trigger=] |
     1. Let |matchingReports| be all entries in the [=event-level report cache=] where all of the following are true:
          * entry's [=event-level report/report time=] and |report|'s [=event-level report/report time=] are equal.
          * entry's [=event-level report/source identifier=] [=string/is=] |report|'s [=event-level report/source identifier=]
-    1. If |matchingReports| is empty, then set |sourceToAttribute|'s [=attribution source/event-level attributable=] value to false
-        and return <strong>dropped</strong>.
+    1. If |matchingReports| is empty:
+        1. Set |sourceToAttribute|'s [=attribution source/event-level attributable=] value to false.
+        1. Let |debugData| be the result of running [=obtain debug data on trigger registration=]
+            with "`trigger-event-excessive-reports`", |trigger|, |sourceToAttribute| and |report|.
```

Comment:
If there are no matching reports we want to return `excessive-reports`?

Reply:
If there are no matching reports, then that means the number of event-level reports have been reached, and therefore we return `excessive-reports`.